controllers/__init__.py

```diff
@@ -7,14 +7,24 @@
 
 import pymongo
 from pymongo.collection import ReturnDocument
+from pymongo.errors import AutoReconnect
 from pymongo.operations import UpdateOne
+from tenacity import retry_if_exception_type, stop_after_attempt, wait_random
 
 from kytos.core import log
 from kytos.core.db import Mongo
 from napps.kytos.topology.db.models import (InterfaceDetailDoc, LinkDoc,
                                             SwitchDoc)
+from napps.kytos.topology.retry import before_sleep, for_all_methods, retries
 
 
+@for_all_methods(
+    retries,
+    stop=stop_after_attempt(3),
+    wait=wait_random(min=0.1, max=1),
```

Comment:
I wonder if these values shouldn't be configurable.

Reply:
Great idea @ajoaoff, since the retries decorators will eventually be moved to the core, it's probably worth using environment variables here, what do you think of using these env variables with these default values? [On kytos core db module](https://github.com/kytos-ng/kytos/blob/master/kytos/core/db.py#L53-L55), we've already have some `MONGO_*` env vars so it would follow that idea:

`MONGO_AUTO_RETRY_STOP_AFTER_ATTEMPT=3` 
`MONGO_AUTO_RETRY_WAIT_RANDOM_MIN=0.1`
`MONGO_AUTO_RETRY_WAIT_RANDOM_MAX=1`